src/index.html

```diff
@@ -17,16 +17,31 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        action="post"
+        data-qa="keypress"
+      >
+        <input
+          class="inputs__bigInput"
+          type="text"
+          data-qa="big"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+      <form
+        class="inputs__form"
+        action="post"
```

Comment:
Similar to the previous form, the `action` attribute should contain a URL or path to a server endpoint. Use the `method` attribute with the value `post` if you intended to specify the HTTP method.